strings/base_implements.h

```diff
@@ -1033,8 +1033,8 @@ namespace winrt::impl
                     {
                         return error_bad_alloc;
                     }
-                    *array = std::copy(local_iids.second, local_iids.second + local_count, *array);
-                    std::copy(inner_iids.cbegin(), inner_iids.cend(), *array);
+                    auto _array = std::copy(local_iids.second, local_iids.second + local_count, *array);
```

Comment:
[nit] I think the name `_array` with the leading underscore is misaligned with the naming pattern in this code base.  Something like `array_copy` or `local_array` would probably fit better.